nilearn/datasets/tests/test_func.py

```diff
@@ -111,41 +113,80 @@ def test_fetch_haxby(tmp_path, request_mocker):
 def _adhd_example_subject(match, request):
     contents = [
         Path("data", match.group(1), match.expand(r"\1_regressors.csv")),
-        Path("data", match.group(1),
-             match.expand(r"\1_rest_tshift_RPI_voreg_mni.nii.gz"))
+        Path(
+            "data",
+            match.group(1),
+            match.expand(r"\1_rest_tshift_RPI_voreg_mni.nii.gz"),
+        ),
     ]
     return list_to_archive(contents)
 
 
 def _adhd_metadata():
     sub1 = [3902469, 7774305, 3699991]
-    sub2 = [2014113, 4275075, 1019436,
-            3154996, 3884955, 27034,
-            4134561, 27018, 6115230,
-            27037, 8409791, 27011]
-    sub3 = [3007585, 8697774, 9750701,
-            10064, 21019, 10042,
-            10128, 2497695, 4164316,
-            1552181, 4046678, 23012]
-    sub4 = [1679142, 1206380, 23008,
-            4016887, 1418396, 2950754,
-            3994098, 3520880, 1517058,
-            9744150, 1562298, 3205761, 3624598]
+    sub2 = [
+        2014113,
+        4275075,
+        1019436,
+        3154996,
+        3884955,
+        27034,
+        4134561,
+        27018,
+        6115230,
+        27037,
+        8409791,
+        27011,
+    ]
+    sub3 = [
+        3007585,
+        8697774,
+        9750701,
+        10064,
+        21019,
+        10042,
+        10128,
+        2497695,
+        4164316,
+        1552181,
+        4046678,
+        23012,
+    ]
+    sub4 = [
+        1679142,
+        1206380,
+        23008,
+        4016887,
+        1418396,
+        2950754,
+        3994098,
+        3520880,
+        1517058,
+        9744150,
+        1562298,
+        3205761,
+        3624598,
+    ]
     subs = pd.DataFrame({"Subject": sub1 + sub2 + sub3 + sub4})
     return dict_to_archive(
-        {"ADHD200_40subs_motion_parameters_and_phenotypics.csv":
-         subs.to_csv(index=False)})
+        {
+            "ADHD200_40subs_motion_parameters_and_phenotypics.csv": subs.to_csv(  # noqa E501 line too long and black will not fix it
```

Comment:
alternatively you can put the filename in a variable

Reply:
good idead